src/js/components/Avatar/Avatar.js

```diff
@@ -15,10 +16,12 @@ const Avatar = ({
   justify = 'center',
   round = 'full',
   size = 'medium',
+  messages,
```

Comment:
we would need to add `messages` prop to index.d.ts file as well as proptypes in Avatar and Grommet